parsec/core/fs/remote_loader.py

```diff
@@ -101,7 +96,7 @@ def __init__(
 
                 # Make sure author had the right to do this
                 existing_user_role = current_roles.get(unsecure_certif.user_id)
-                if not current_roles and unsecure_certif.user_id == author.user_id:
+                if not current_roles and unsecure_certif.user_id == author.device_id.user_id:
```

Comment:
In some other parts of the code base, `author` refers to a `DeviceID`. Maybe we should have some naming convention to differentiate a `DeviceID` from a `VerifiedRemoteDevice`.

Reply:
I also noticed:
```
± git grep "remote_device_manager" | wc
     11      40    1100
± git grep "remote_devices_manager" | wc
     55     235    5625
```
This is going to bite us for sure :)

Reply:
> This is going to bite us for sure :)

Yep, pretty sure we need a cleanup there